### PR TITLE
[Site] Chapters Page Implementation 

### LINKED ISSUE #64: [Site] Add Chapters of the book page
**Current State of the Service Mesh Patterns site**

Chapters of the book on the site need to be implemented .

**Desired State of the Service Mesh Patterns site**

---
### Contributor Resources
This website uses React and GitHub Pages. Site content is found under the [`master` branch](https://github.com/service-mesh-books/service-mesh-patterns/tree/master/src).
- See the [Contributing to Service Mesh Patterns Website](https://github.com/service-mesh-patterns/service-mesh-patterns/#readme) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- Layer5 [Brand Kit](https://layer5.io/company/brand)
- 📑 [Service Mesh Patterns in Meshery Catalog](https://meshery.io/catalog)
- 📖 [Service Mesh Patterns Book](https://layer5.io/learn/service-mesh-books/service-mesh-patterns)
- 🎨 [Site designs in Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=5310%3A0). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Jonathan-Foo**
Signed-off-by: Jonathan Foo <jonathan.foozh@gmail.com>

**Description**
Implemented the Chapters page according to Figma design.

This PR fixes #64 

image

Open Accordion:
image

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1f7b362bc54b7ba3d4e980def09574807f5cd13c at: https://632c7fe32cf1865482a9f1f0--service-mesh-patterns.netlify.app

**Comment by Jonathan-Foo** [comment]
Responsive Mobile View:
image

File: site/src/pages/chapters.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ChaptersPage from "../sections/Chapters";
+
+const Chapters = () => {
+  return (
+    <>
+      <ChaptersPage />
+    </>
```
Comment (Nikhil-Ladha) [inline site/src/pages/chapters.js:8]: Add the `SeoComponent`, `Layout` and the `Footer` similar to what we have in the [index](https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/site/src/pages/index.js) file

File: site/src/sections/Chapters/Accordion.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+              <p>{topic.type}</p>
+            </div>
+          )) }
+        </div>
+      )}
+    </>
+  );
+};
+
+export default Accordion;
```
Comment (Nikhil-Ladha) [inline site/src/sections/Chapters/Accordion.js:30]: The `Accordion` can be moved to the component folder.
P.S: Thanks for making it generic :)

File: site/src/sections/Chapters/Callout.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+          <p>JOIN THE CONVERSATION</p>
+        </div>
+        <p>Join the Discussion Forum</p>
+        <img src={layer5Logo} alt="layer5 logo" />
+      </a>
+    </div>
+  );
+};
+
+export default Callout;
```
Comment (Nikhil-Ladha) [inline site/src/sections/Chapters/Callout.js:20]: Can be moved to the `component` folder

File: site/src/sections/Chapters/Callout.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import personIcon from "../../assets/images/chapters/person.svg";
+import layer5Logo from "../../assets/images/chapters/layer5-discuss-white.png";
+
+const Callout = () => {
+  return (
+    <div className="callout">
+      <a href="https://discuss.layer5.io/">
```
Comment (Nikhil-Ladha) [inline site/src/sections/Chapters/Callout.js:8]: Add a `target="_blank"` and `rel=noreferrer`

**Review by Nikhil-Ladha — COMMENTED** [review]
Nailed it! 🔨 
Just a couple of minor adjustments needed, rest looks great 💯

File: site/src/pages/chapters.js (hunk 1 above)
Comment (Jonathan-Foo) [reply site/src/pages/chapters.js:8]: Do I need to include the schema from the index page as well?

File: site/src/sections/Chapters/Callout.js (hunk 4 above)
Comment (Jonathan-Foo) [reply site/src/sections/Chapters/Callout.js:8]: Resolved

File: site/src/sections/Chapters/Callout.js (hunk 3 above)
Comment (Jonathan-Foo) [reply site/src/sections/Chapters/Callout.js:20]: Resolved

File: site/src/sections/Chapters/Accordion.js (hunk 2 above)
Comment (Jonathan-Foo) [reply site/src/sections/Chapters/Accordion.js:30]: No worries and done!

**Comment by l5io** [comment]
🚀 Preview for commit eed973bf856ba5d803397cfe5c45c98203cf5b94 at: https://632d0890d8392f0e26934ee9--service-mesh-patterns.netlify.app

**Comment by Nikhil-Ladha** [comment]
On last itme, the "Open-Close" button is not clickable, though the whole Accordion is (which is good). But, the button should be clickable too. Can you please quickly resolve that?

**Comment by Jonathan-Foo** [comment]
Hi @Nikhil-Ladha, the button works now.

**Comment by l5io** [comment]
🚀 Preview for commit c50d2ed6152064855af05f164ef0b64ca4b0cd73 at: https://632d8f00adf64e01bc5a2c37--service-mesh-patterns.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looks great!

**Comment by Jonathan-Foo** [comment]
Thank you! 😄

**Comment by legendarykamal** [comment]
@Jonathan-Foo good work :+1: :medal_sports:

**Pull request merged**